beamer/health/check.py

```diff
@@ -192,27 +210,54 @@ def fetch_events() -> ChainEventMap:
     return cast(ChainEventMap, events)
 
 
-def make_web3(rpc: str) -> Web3:
-    web3 = Web3(Web3.HTTPProvider(rpc, request_kwargs=dict(timeout=5)))
-    web3.middleware_onion.inject(geth_poa_middleware, layer=0)
-    return web3
+def get_transfer_token_symbol(transfer: Transfer, token_deployments: TokenMap) -> str | None:
+    for token_symbol, deployments in token_deployments.items():
+        if any(
+            transfer["created"].source_token_address.lower() == address.lower()
+            and transfer["created"].chain_id == int(chain_id)
+            for chain_id, address in deployments
+        ):
+            return token_symbol
+
+    return None
+
+
+def get_transfer_value_formatted(
+    request: RequestCreated, token_deployments: TokenMap, token_details: TokenDetailsMap
+) -> str:
+    token_source_chain = request.chain_id
+    token_source_address = request.source_token_address
+    token = None
+    transfer_value = "0"
+
+    for token_symbol, deployments in token_deployments.items():
+        if deployments.count([str(token_source_chain), token_source_address]):
+            token = token_details[token_symbol]
+            break
+
+    if token is not None:
+        decimal_value = get_token_amount_in_decimals(request.amount, token)
```

Comment:
When will `token` be `None` here?

Reply:
This will happen when we are trying to get the transfer value for transfers of tokens that are not yet defined inside `health-check-mainnet.conf` under the token definitions.